From 6575ea8356de2fc9b2aeadca2a8fbe60ef8eb0cf Mon Sep 17 00:00:00 2001
From: davijosw <82056592+davijosw@users.noreply.github.com>
Date: Mon, 18 Apr 2022 08:51:45 -0300
Subject: [PATCH] hwdb: Update 60-keyboard.hwdb (#23074)

Fix Fn+F1 (toggle touchpad) that wasn't working.

Closes #23058

(cherry picked from commit 39cd62c30c2e6bb5ec13ebc1ecf0d37ed015b1b8)

Related: #2087778
---
 hwdb.d/60-keyboard.hwdb | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/hwdb.d/60-keyboard.hwdb b/hwdb.d/60-keyboard.hwdb
index 7c830fb9b1..adf5e9ac35 100644
--- a/hwdb.d/60-keyboard.hwdb
+++ b/hwdb.d/60-keyboard.hwdb
@@ -1931,6 +1931,13 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnViewSonic:pnVPAD10:*
  KEYBOARD_KEY_db=prog1                                  # Home button
  KEYBOARD_KEY_dd=back                                   # Second button labeled Back
 
+###########################################################
+# Positivo-Vaio
+###########################################################
+# Vaio FE14
+evdev:name:AT Translated Set 2 keyboard:dmi:bvn*:bvr*:bd*:svnPositivoBahia-VAIO:pnVJFE41F11*:pvr*:*
+ KEYBOARD_KEY_76=f21		                     # Fn+F1 toggle touchpad
+
 ###########################################################
 # Other
 ###########################################################
